src/framework/Templates/TemplateConfigModal/TemplateConfigModal.tsx

```diff
@@ -270,7 +270,7 @@ const BasicTemplateDetails = (props: BasicDetailsInterface): JSX.Element => {
                     </Container>
                     {isGitSyncEnabled && showGitFields && (
                       <GitSyncStoreProvider>
-                        <GitContextForm formikProps={formik} gitDetails={gitDetails} />
+                        <GitContextForm formikProps={formik as any} gitDetails={gitDetails} />
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
                        <GitContextForm formikProps={formik as unknown} gitDetails={gitDetails} />
```